http4s-client/src/main/scala/com/avast/server/toolkit/http4s/Http4sClientModule.scala

```diff
@@ -0,0 +1,62 @@
+package com.avast.server.toolkit.http4s
+
+import cats.effect.{ConcurrentEffect, Resource, Sync}
+import cats.implicits._
+import cats.{Applicative, Traverse}
+import com.avast.server.toolkit.ssl.{SslContextConfig, SslContextModule}
+import javax.net.ssl.SSLContext
+import org.http4s.client.Client
+import org.http4s.client.blaze.BlazeClientBuilder
+
+import scala.concurrent.ExecutionContext
+import scala.language.higherKinds
+
+/** Provides single initialized HTTP client and a method to create more of them. */
+class Http4sClientModule[F[_]: ConcurrentEffect](val client: Client[F]) {
```

Comment:
`class Http4sClientModule` doesn't add any value. Having `object Http4sClientModule` with `make` is enough.